scripts/analyze.sh

```diff
@@ -1,26 +1,45 @@
 #!/usr/bin/env bash
 
 function run_analyze() {
-  flutter pub get
+  if ! grep -q "dart_code_metrics" ./pubspec.yaml; then
+    echo "Missing dart_code_metrics in pubspec.yaml"
+    exit 1
+  fi
+
+  result=$(flutter pub get 2>&1) # Sadly a pub get can block up our actions as it will retry forever if a package is not found, but this should atleast report everything else.
```

Comment:
I don't understand why discarding the output would prevent it from "retrying forever" and getting our action stuck, seems it would just get stuck but w/o any indication as to why then